indico/web/templates/forms/protection_message_widget.html

```diff
@@ -0,0 +1,31 @@
+{% extends 'forms/base_widget.html' %}
+
+{% block html -%}
+    {{ field.radio_widget(field, **input_args)  }}
+    {{ field.render_protection_message() }}
+{%- endblock %}
+
+{% block javascript -%}
+    <script>
+        (function() {
+            'use strict';
+
+            $('[id^={{ field.id }}]').on('change', function() {
```

Comment:
maybe including the name in those selectors too would be good `[name=...][id^=..]` - otherwise you could have (rare) cases where it matches too much